pluto-m2k/config/m2k/m2k_part1.py

```diff
@@ -0,0 +1,486 @@
+import libm2k
+import subprocess
+from time import sleep
+import math
+import numpy as np
+from datetime import datetime
+import sys
+
+SHOW_TIMESTAMP = True
+SHOW_START_END_TIME = True
+MAX_HIGH_GAIN = 2.515
+MIN_HIGH_GAIN = 2.465
+MAX_LOW_GAIN = 10.15
+MIN_LOW_GAIN = 9.65
+ADC_CONST_ERR_THRESHOLD = 0.1
+M2KCALIB_INI = "m2k-calib-factory.ini"
+M2KCALIB_INI_LOCAL = "/tmp/" + M2KCALIB_INI
+PWS_POS_FIRST = 0.1;
+PWS_POS_SECOND = 4.5;
+PWS_NEG_FIRST = -PWS_POS_FIRST;
+PWS_NEG_SECOND = -PWS_POS_SECOND;
+
+#Default PWS calibration values
+OFFSET_POS_DAC = 0
+GAIN_POS_DAC = 1
+OFFSET_POS_ADC = 0
+GAIN_POS_ADC = 1
+OFFSET_NEG_DAC = 0
+GAIN_NEG_DAC = 1
+OFFSET_NEG_ADC = 0
+GAIN_NEG_ADC = 1
+
+#*********************************************************************************************************
+#	STEP 5
+#*********************************************************************************************************
+
+def _osc_change_gain_mode(ch, high):
+	global osc
+	osc.enableChannel(ch, True)
+
+	if high:
+		osc.setRange(ch, libm2k.HIGH_GAIN)
+	else:
+		osc.setRange(ch, libm2k.LOW_GAIN)
+
+
+def _osc_check_range(high, value):
+	if high:
+		if (value < MAX_HIGH_GAIN) and (value > MIN_HIGH_GAIN):
+			return True
+		else:
+			return False
+	else:
+		if (value < MAX_LOW_GAIN) and (value > MIN_LOW_GAIN):
+			return True
+		else:
+			return False
+
+def disable_ref_measurement():
+	subprocess.run(["./ref_measure_ctl.sh", "disable"])
+	#extern.start("sshpass -pjig ssh jig@localhost sudo ~/plutosdr-m2k-production-test-V2/ref_measure_ctl.sh disable");
```

Comment:
is this commented code required ?

Reply:
no, I will update the commits to remove the unnecessary comments